src/components/DiagramFrame/MetadataPanel/common.tsx

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Here's the heart of the change. 

Applying styles for things that don't have defaultProps. Using an FC wrapper when no styles are needed

Reply:
Common as a name is a bit generic. Consider

utility_components
common_components
shared_components
metadata_components